kong/llm/drivers/anthropic.lua

```diff
@@ -209,8 +209,7 @@ local function handle_stream_event(event_t, model_info, route_type)
     and event_data.usage then
       return nil, nil, {
         prompt_tokens = nil,
-        completion_tokens = event_data.meta.usage
-                        and event_data.meta.usage.output_tokens
+        completion_tokens = event_data.usage.output_tokens
```

Comment:
This seems more than just syntax correction, eh? Can you give a hint why you did this?

Reply:
@jschmid1 

It is a mistake :(

One of the earlier "event types" has data.meta.usage, whereas this one has data.usage, for its token reporting.

Not entirely sure why they decided to use different messaging formats for similar things, but when I ctrl+cv'd it I didn't notice they were different.

Reply:
Sounds like we'd benefit from an additional test here :)

Reply:
It will be covered in 3.8 real integration tests, right now we have Insomnia suite running through everything.